snap/snapcraft.yaml

```diff
@@ -1,6 +1,6 @@
 name: minio
-version: git
-summary: open source object storage server compatible with Amazon S3 APIs
+version: master
```

Comment:
Done.
btw, one item in our backlog is to move all the build directories inside the snap directory, to make this clearer.